doc/nrf/protocols/bt/bt_mesh/reserved_ids.rst

```diff
@@ -17,59 +17,67 @@ Reserved vendor model IDs
-   | Simple OnOff Set Unreliable       | 0x03   |
-   +-----------------------------------+--------+
-   | Simple OnOff Status               | 0x04   |
-   +-----------------------------------+--------+
-   | Rssi Ack                          | 0x05   |
-   +-----------------------------------+--------+
-   | Rssi Send Data                    | 0x06   |
-   +-----------------------------------+--------+
-   | Rssi Util Request Database Beacon | 0x07   |
-   +-----------------------------------+--------+
-   | Rssi Util Send Database Beacon    | 0x08   |
-   +-----------------------------------+--------+
-   | Thingy52 RGB Set                  | 0x09   |
-   +-----------------------------------+--------+
-   | Chat Message                      | 0x0A   |
-   +-----------------------------------+--------+
-   | Chat Private Message              | 0x0B   |
-   +-----------------------------------+--------+
-   | Chat Message Reply                | 0x0C   |
-   +-----------------------------------+--------+
-   | Chat Presence                     | 0x0D   |
-   +-----------------------------------+--------+
-   | Chat Presence Get                 | 0x0E   |
-   +-----------------------------------+--------+
+   +------------------------------------+--------+
+   | Message name                       | Opcode |
+   +====================================+========+
+   | Simple OnOff Set                   | 0x01   |
+   +------------------------------------+--------+
+   | Simple OnOff Get                   | 0x02   |
+   +------------------------------------+--------+
+   | Simple OnOff Set Unreliable        | 0x03   |
+   +------------------------------------+--------+
+   | Simple OnOff Status                | 0x04   |
+   +------------------------------------+--------+
+   | Rssi Ack                           | 0x05   |
+   +------------------------------------+--------+
+   | Rssi Send Data                     | 0x06   |
+   +------------------------------------+--------+
+   | Rssi Util Request Database Beacon  | 0x07   |
+   +------------------------------------+--------+
+   | Rssi Util Send Database Beacon     | 0x08   |
+   +------------------------------------+--------+
+   | Thingy52 RGB Set                   | 0x09   |
+   +------------------------------------+--------+
+   | Chat Message                       | 0x0A   |
+   +------------------------------------+--------+
+   | Chat Private Message               | 0x0B   |
+   +------------------------------------+--------+
+   | Chat Message Reply                 | 0x0C   |
+   +------------------------------------+--------+
+   | Chat Presence                      | 0x0D   |
+   +------------------------------------+--------+
+   | Chat Presence Get                  | 0x0E   |
+   +------------------------------------+--------+
+   | Distance Measurement Server Opcode | 0x0F   |
+   +------------------------------------+--------+
+   | Distance Measurement Client Opcode | 0x10   |
```

Comment:
These two seem to be the only ones that have Opcode mentioned in the name. Just checking if they should have this, or if it should be removed?